Project-1/README.md

```diff
@@ -38,29 +102,234 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
   (d)  O that says that x is an only child
   (e)  T that says that x has exactly two brothers 
 ```
+⊔ ⊓ ⊧ ⊭ ⊦ ⊬ ⊏ ⊐ ⊑ ⊒ C ¬ ≡ ≠ ≥ ≤
+
+   First attempts
```

Comment:
All answers should be "Cannot be done since ALC isn't expressive enough".

Reply:
Corrections made!